### ISSUE TITLE
wallet with 2fa  the seed in wallet menu does not shown

### ISSUE THREAD

**Issue by johnnydlgs** [spesmilo/electrum#9096]
wallet with 2fa  the seed in wallet menu does not shown while in other wallets with no 2fa can be used. How i recover seed (lost) in that case ? ,( i have private keys etc ) .is that a bug?

**Comment by johnnydlgs** [comment]
![Screenshot_2024-06-17_13-08-33](https://github.com/spesmilo/electrum/assets/106869764/d5eb6ea4-e4aa-48ab-bd85-8a8390c9d58b)
every transaction i do no matter the amount or the receiver wallet i receive this error message and in android something similar too . do i do something wrong , or is a bug?

**Comment by ecdsa** [comment]
which version of electrum are you using?

**Comment by johnnydlgs** [comment]
thanks for the fast response , version 4.5.5  both in linux 6.6.15 and in android 13

**Comment by SomberNight** [comment]
- Is this a new 2fa wallet with some existing history, or is this the first time you are trying to spend from it?
- If you click the blue shield icon in the bottom right (on desktop GUI), in the bottom of the dialog, what does it say re how many "prepaid transactions" do you have?
- Also, how does the bitcoin amount you are trying to send compare to the fee (mining fee, and trustedcoin fee)? As in, are you trying to send just 1-2 mBTC (or even less), or is it a higher amount?

**Comment by johnnydlgs** [comment]
Its the all first transaction for this wallet thus i have 0 transactions and the amount is  5 and more mBTC and i have lower the first fee amount accordingly which initially showed me that there were no enough funds. It may has something with the amount as it happens in android as well . I would try to add more from another wallet , but as the only 2fa wallet and already caused me those difficulties dare to say , i am reluctant to do that so firstly i am trying to move the amount to another wallet . Is that normal that the indication of the seed in the 2fa wallet is not working ?

**Comment by SomberNight** [comment]
> Is that normal that the indication of the seed in the 2fa wallet is not working ?

Yes, that part is normal. The seed is not stored in the wallet for 2fa wallets, as the seed alone is enough to spend to coins, hence it allows bypassing the second factor.

**Comment by SomberNight** [comment]
>> Also, how does the bitcoin amount you are trying to send compare to the fee (mining fee, and trustedcoin fee)? As in, are you trying to send just 1-2 mBTC (or even less), or is it a higher amount?
>
> the amount is 5 and more mBTC

But on your latest screenshot, the amount is below 1 mBTC. Could you please share a screenshot that shows the output amounts?
E.g. from the "Confirm payment" screen, see below:
<details>
 <summary>pic</summary>

![pic1](https://github.com/spesmilo/electrum/assets/29142493/c0a0ca08-f01e-4a4e-976f-0753cce3db30)
</details>

I think the issue might be that the amount is too low to prepay for the 2FA (Trustedcoin) fee for the next batch of transactions.

**Comment by johnnydlgs** [comment]
the amount was 0.0001 for one time i used but actually the error was due to miscommunication or wrongly infos /connection  from the thustedcoin server as it is just worked with the same effort and the  max amount witch previously didn"t ... Thank you very much for the effort and my excuses as well , i will just delete the wallet because of  lack of the seed  anyway
